modeling/model_utils/modeling.py

```diff
@@ -0,0 +1,80 @@
+import logging.config
+from typing import List, Dict
+
+import pandas as pd
+from sklearn.model_selection import (
+    train_test_split,
+    cross_validate,
+    GridSearchCV,
+)
+from sklearn.pipeline import Pipeline
+
+from model_utils.constants import LOG_CONFIG_PATH
+
+logging.config.fileConfig(fname=LOG_CONFIG_PATH, disable_existing_loggers=False)
+logger = logging.getLogger(__name__)
+
+
+def load_data(read_path: str, X_col: List[str], y_col: str):
+    logger.info(f"Loading data from {read_path}")
+    X = pd.read_csv(read_path, usecols=X_col)
+    y = pd.read_csv(read_path, usecols=[y_col])
+    return X, y
+
+
+def split_data(X: pd.DataFrame, y: pd.DataFrame):
+    logger.info(f"Splitting data into train and test set")
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.33, random_state=42
```

Comment:
I would prefer to leave the test_size and random_state as variables instead of hard coded... but its's ok here.. 
Just to say I leave 0.2 as dev set, also random_state = 0. We need to keep the same dev set..

Reply:
i was thinking of hard coding it so everyone who uses this pipeline will be using the same test size and random seed and it will be easier for us to compare performance across different pipelines

Reply:
i have changed it to 0.2 and set random seed as 0 :)